iothub/device/src/DeviceClient.ConventionBasedOperations.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Azure.Devices.Client
     public partial class DeviceClient : IDisposable
     {
         /// <summary>
-        /// The <see cref="PayloadConvention"/> that the client uses for convention-based operations.
```

Comment:
No need to reference the type we are, as the web docs will automatically link to types.